wgpu-rs/src/lib.rs

```diff
@@ -233,6 +233,29 @@ impl<'a> TextureCopyView<'a> {
     }
 }
 
+pub struct CreateBufferMapped<'a, T> {
+    id: wgn::BufferId,
+    data: &'a mut [T],
+}
+
+impl<'a, T> CreateBufferMapped<'a, T>
+        where T: Copy {
+    pub fn from_slice(self, slice: &[T]) -> Buffer {
```

Comment:
typically `from_xx` is used for constructors. Let's rename this to `fill_from_slice` or something like this?